src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      action="#"
+      method="POST"
+      class="form"
+      data-qa="big"
+    >
+      <input
+        name="searchInput"
+        class="form__input"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+      >
+      <button type="submit" class="form__btn"></button>
+    </form>
+
+    <form
+      action="#"
+      method="POST"
+      class="form"
+      data-qa="small"
+    >
+      <input
+        name="searchInput"
+        class="form__input"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+      <button type="submit" class="form__btn"></button>
```

Comment:
I see you are adding a search icon for submit button.
Usually search icons in this kind of forms are not submit buttons.
Please try to change this, and use different tag for adding search icon. You can try with **img** and **label**.